layouts/opensource/single.html

```diff
@@ -55,7 +55,10 @@ <h3 class="font-bold text-xl sm:text-2xl lg:text-3xl">{{ .title }}</h3>
                                 <h4 class="font-bold text-lg">{{ .name }}</h4>
                                 <p class="mt-4">{{ .description }}</p>
                             </div>
-                            <img src="{{ .image }}" alt="" class="lg:ml-4 lg:mt-0 mt-6 h-96">
+                            <<<<<<< HEAD <img src="{{ .image }}" alt="" class="lg:ml-4 lg:mt-0 mt-6">
+                                =======
+                                <img src="{{ .image }}" alt="" class="lg:ml-4 lg:mt-0 mt-6 h-96">
```

Comment:
Should this line also be like line 32 and display the alt text like alt="{{.alt}}"?